lib/schema.js

```diff
@@ -138,6 +138,25 @@ function Schema(obj, options) {
 
       next();
     });
+
+    function genUpdates() {
```

Comment:
This'll break in strict mode. Use `var genUpdates = function() {};` instead please :)